### ISSUE TITLE
FR: Add Glide Parameters

### ISSUE THREAD

**Issue by aerni90** [aryehraber/statamic-responsive-img#7]
It would be amazing to have access to the glide parameters. The most common use in my case would be to set the width and height of a picture and use the Statamic focal point feature. Amazing would be to have presets like we have with glide. There you could set everything from width, crop, quality, scaling-rate etc.

**Comment by aryehraber** [comment]
Yeah I totally agree, this was a bit of an oversight tbh... Adding to FR list 😊

**Comment by sxdj** [comment]
Would also be awesome if the original filename could be retained, as being important for SEO. 

Though it's no explicit [Glide param](http://glide.thephpleague.com/1.0/api/quick-reference/), a filename can optionally be supplied after the path element (according to https://github.com/peda/statamic-glide-vanity-url)

Thanks for considering!

**Comment by tobiasvielmetter** [comment]
Is this something that is planned or should we not wait for it? I might otherwise looking into solving this by myself, but can't guarantee that I will be able to deliver a PR.
